xmpp/src/main/java/ch/qos/logback/classic/net/XmppAppender.java

```diff
@@ -0,0 +1,143 @@
+package ch.qos.logback.classic.net;
```

Comment:
Please fix the following issues:
 * Use 4-space tabs
 * Add standard file header